app/models/topic_poster.rb

```diff
@@ -15,8 +15,13 @@ def attributes
     }
   end
 
-  # TODO: Remove when old list is removed
-  def [](attr)
```

Comment:
I'm not sure but is this safe to remove?

Reply:
I'm pretty sure we aren't using it anymore. This TODO has been there since the initial release and I thought it would be a good idea to remove it while being in there. All the functionality I tested is still working without this function.